app/services/bulk_update/recommendations_uploads/validate_trainee.rb

```diff
@@ -36,13 +36,13 @@ def find!
       def validate!
         return if trainee
         # if no singular trainee in state trn_recevied was found then check if there are multiple in trn_received
-        return @messages << error_message(:multiple_trainees_trn_recieved, found_with:) if trainees.size > 1
+        return @messages << error_message(:multiple_trainees_trn_received, count: trainees.count, found_with: found_with) if trainees.size > 1
         # otherwise check if multiple in non-draft non-trn-received were found
-        return @messages << error_message(:multiple_trainees_found_via, found_with:) if trainees_not_trn_received.size > 1
+        return @messages << error_message(:multiple_trainees_found_via, count: trainees.count, found_with: found_with) if trainees_not_trn_received.size > 1
         # otherwise check if one was found outside of non-draft, non-trn_received
         return @messages << error_message(:trainee_wrong_state, state: trainee_not_trn_received.state.humanize) if trainee_not_trn_received
 
-        @messages << error_message(:no_trainee_matched)
+        @messages << error_message(:no_trainee_matched, id_available: id_columns)
```

Comment:
Can we add a test example to cover the added information?

Reply:
Should be in there now